app/build.gradle

```diff
@@ -353,7 +353,7 @@ distributions {
                 exclude 'es.jar'
                 exclude 'es-*.jar'
             }
-            into("lib") {
+            into("lib/main") {
```

Comment:
@mfussenegger This is one way of changing how we define the classpath. We want the Crate main jar to be the first element of the classpath but we're adding it twice if we use `lib/*` afterwards. Of course we could use something like `find lib --exclude "crate*.jar" but IMHO separating the directories is a cleaner solution.